src/stdatamodels/jwst/datamodels/tests/test_schema_against_crds.py

```diff
@@ -196,9 +197,14 @@ def test_crds_selectors_vs_datamodel(jail_environ, instrument):
 
     # get the reftypes
     reftypes = imap.get_filekinds()
+
     # remove pars- files
     _ = [reftypes.remove(name) for name in reftypes[::-1] if name.startswith(ignored_stems)]
 
+    # remove known missing
```

Comment:
should we go ahead and remove this here before merging, now that this reftype is recognized, or does the removal only make sense once Jane's PR is merged?

Reply:
The second one is correct. Removing it only makes sense once Jane's PR is merged. Otherwise the test would fail when it encountered the 'psf' reftype.